root-packages/vnstat/build.sh

```diff
@@ -0,0 +1,8 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/vergoh/vnstat
+TERMUX_PKG_DESCRIPTION="A console-based network traffic monitor"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@Denzy7"
+TERMUX_PKG_VERSION=2.12
+TERMUX_PKG_SRCURL=https://github.com/vergoh/vnstat/releases/download/v${TERMUX_PKG_VERSION}/vnstat-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=b7386b12fc1fc6f47fab31f208b12eda61862e63e229e84e95a6fa80406d2852
+TERMUX_PKG_DEPENDS="libsqlite"
```

Comment:
We'll probably also want a `TERMUX_PKG_SERVICE_SCRIPT` for a `sv` service for the daemon.

Should be able to interpolate from the upstream Systemd service example.
https://github.com/vergoh/vnstat/blob/19bacde8d7123c4184fcc4b2926aab0e1ada2d6d/examples/systemd/simple/vnstat.service#L6-L9

Reply:
Is `sv` run as root?

Reply:
No its run entirely in userspace.
It should suffice to set:
```suggestion
TERMUX_PKG_DEPENDS="libsqlite"
TERMUX_PKG_SERVICE_SCRIPT=('vnstatd' 'vnstatd -n')
```
The format is: TERMUX_PKG_SERVICE_SCRIPT=(`daemon-name` `daemon-command`)

Reply:
Nice. I guess I'll also have to copy [how docker cleans up after sv down](https://github.com/termux/termux-packages/blob/master/root-packages/docker/build.sh#L127-L137)